components/system/components/Table.js

```diff
@@ -127,7 +140,24 @@ export class Table extends React.Component {
 
           return (
             <React.Fragment key={`${r.id}-${i}`}>
-              <div css={STYLES_TABLE_ROW}>
+              <div
+                css={STYLES_TABLE_ROW}
+                onClick={() => this._handleClick(r.id)}
+                style={{
+                  cursor:
+                    this.props.onClick && r.children ? "pointer" : "default",
+                }}
+              >
+                {this.props.onClick && r.children ? (
+                  <SVG.ChevronDown
```

Comment:
* **Future enhancement**
* **Non-blocking**

I worry that the hitbox for this SVG is going to be annoying, can we wrap this in a rect element that ensures the hitbox